app/components/find_interface/phase_banner/view.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module FindInterface
+  class PhaseBanner::View < ViewComponent::Base
+    include ViewHelper
+    def initialize(no_border: false)
+      super
+      @no_border = no_border
+    end
+
+    def environment_label
+      Settings.environment.label
+    end
+
+    def environment_colour
+      {
+        "development" => "grey",
+        "qa" => "orange",
+        "review" => "purple",
+        "sandbox" => "purple",
+        "staging" => "red",
+        "unknown-environment" => "yellow",
```

Comment:
Can we add `load-test`

Reply:
What colour is that?

Reply:
Discussed offline, decided to leave this here as its part of other ongoign work.